docs/sphinx/manuals/roc_send.rst

```diff
@@ -37,6 +37,7 @@ Options
 --resampler-window=INT      Number of samples per resampler window
 --interleaving              Enable packet interleaving  (default=off)
 --poisoning                 Enable uninitialized memory poisoning (default=off)
+--poisoning                 Enable uninitialized memory poisoning (default=off)
```

Comment:
A typo: should be --profiling.

BTW, you can generate this piece of RST using this command:

```
roc-send -h | scripts/help2rst.py
```